packages/cli/src/module-resolver.ts

```diff
@@ -232,6 +255,23 @@ export function getFileSystemFromTypeScript(
   };
 }
 
+/**
+ * Get the module type for a given package specifier.
+ *
+ * @param packageSpecifier - The specifier for the package.
+ * @param system - The TypeScript system.
+ * @param parentUrl - The URL of the parent module.
+ * @returns The module type for the package.
+ */
+export function getModuleType(
+  packageSpecifier: string,
+  system: System,
+  parentUrl: string,
+) {
+  const resolution = resolveModule(packageSpecifier, parentUrl, system);
```

Comment:
Seems like this can now return `directory`. Is that accounted for everywhere it is used?